train.py

```diff
@@ -90,15 +171,18 @@
     "eo_loss": mse.loss.__class__.__name__,
     "device": device,
     "logging_dir": logging_dir,
-    # **args,
-    # **model_kwargs,
+    **args,
+    **model_kwargs,
 }
 
+if wandb_enabled:
+    wandb.config.update(training_config)
+
 lowest_validation_loss = None
 best_val_epoch = 0
 training_step = 0
 num_validations = 0
-dataloader_length = df.shape[0]
+dataloader_length = train_df.shape[0]
```

Comment:
Shouldn't this be in batches? Cfr https://github.com/WorldCereal/presto-worldcereal/blob/304b2fb33311e50085e2143af9e659322cdd6abd/train.py#L194-L202

Reply:
I think indeed I also computed the `dataloader_length` by dividing the the dataframe length by the batch size.

Reply:
doh! nice catch. Fixed in https://github.com/WorldCereal/presto-worldcereal/pull/8